src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -299,9 +307,10 @@ public void deleteRoles(final Node node) throws RepositoryException {
                     child.getTriples(translator, PropertiesRdfContext.class)
                          .filter(p -> isAclPredicate.test(model.asStatement(p).getPredicate()))
                          .forEachRemaining(t -> {
-                            aclTriples.putIfAbsent(t.getPredicate().getURI(), new ArrayList<>());
+                             aclTriples.putIfAbsent(t.getPredicate().getURI(), new ArrayList<>());
                              if (t.getObject().isURI()) {
-                                aclTriples.get(t.getPredicate().getURI()).add(t.getObject().getURI());
+                                 aclTriples.get(t.getPredicate().getURI()).add(
+                                     substringBeforeLast(t.getObject().getURI(), "/" + JCR_CONTENT));
```

Comment:
Obviously it is a little frightening that outside of the kernel we are seeing "jcr:content". It appears to work... but is unsettling nonetheless.